src/components/20-molecules/datepicker/index.js

```diff
@@ -324,6 +336,39 @@ class AXADatepicker extends NoShadowDOM {
       });
     };
 
+    const versionedHtml = tagName => (strings, ...args) => {
+      const aVersion = AXADatepicker.version[1][tagName];
+      const oldTag = (closing = '') => `<${closing}${tagName}`;
+      const newTag = closing => `${oldTag(closing) + '-' + aVersion}`;
```

Comment:
What do you think to have this version-logic at a centered place? "TAGNAME-VERSION" is located here and in `with-react.js`.

Reply:
yes, that will be factored out in a DRY fashion.